Controller/PaylineController.php

```diff
@@ -55,18 +62,26 @@ public function __construct(EventDispatcherInterface $eventDispatcher, WebGatewa
         $this->defaultErrorUrl = $defaultErrorUrl;
     }
 
+    /**
+     * @param Request $request
+     * @return Response
+     */
     public function paymentNotificationAction(Request $request)
     {
-        $result = $this->payline->verifyWebTransaction($request->get('token'));
+        $result = $this->payline->verifyWebTransaction($request->get('paylinetoken'));
```

Comment:
As we don't exactly know which version is deployed for every customer, I suggest the following:

```php
$result = $this->payline->verifyWebTransaction($request->get('paylinetoken', $request->get('token')));
```

Reply:
:+1: